client/src/components/settings/SettingsList.vue

```diff
@@ -3,113 +3,149 @@
 <template>
   <div class="settings-list-container">
     <ion-list class="settings-list">
-      <!-- change lang -->
-      <settings-option
-        :title="'SettingsModal.language.label'"
-        :description="'SettingsModal.language.description'"
-      >
-        <ms-dropdown
-          class="dropdown"
-          :options="languageOptions"
-          :default-option-key="I18n.getLocale()"
-          @change="changeLang($event.option.key)"
-        />
-      </settings-option>
-      <!-- change theme -->
-      <!-- TODO: REMOVE "disabled=true' : " WHEN DARK MODE WILL BE HERE: https://github.com/Scille/parsec-cloud/issues/5427 -->
-      <settings-option
-        :title="'SettingsModal.theme.label'"
-        :description="'SettingsModal.theme.description'"
-      >
-        <ms-dropdown
-          class="dropdown"
-          :options="themeOptions"
-          :default-option-key="config.theme"
-          @change="changeTheme($event.option.key)"
-          :disabled="true"
-        />
-      </settings-option>
-      <!-- minimize in status bar -->
-      <settings-option
-        v-if="isPlatform('electron') && !isMacOS()"
-        :title="'SettingsModal.minimizeToSystemTray.label'"
-        :description="'SettingsModal.minimizeToSystemTray.description'"
-      >
-        <ion-toggle v-model="config.minimizeToTray" />
-      </settings-option>
-      <!-- MacOS-only toggle confirm -->
-      <settings-option
-        v-if="isPlatform('electron') && isMacOS()"
-        :title="'SettingsModal.confirmBeforeQuit.label'"
-        :description="'SettingsModal.confirmBeforeQuit.description'"
-      >
-        <ion-toggle v-model="config.confirmBeforeQuit" />
-      </settings-option>
-      <!-- send error report -->
-      <settings-option
-        :title="'SettingsModal.enableTelemetry.label'"
-        :description="'SettingsModal.enableTelemetry.description'"
-      >
-        <ion-toggle v-model="config.enableTelemetry" />
-      </settings-option>
-      <settings-option
-        :title="'SettingsModal.seeLogs.label'"
-        :description="'SettingsModal.seeLogs.description'"
-      >
-        <ion-button
-          @click="openLogs"
-          fill="clear"
-          class="see-logs-button"
+      <div class="settings-list-group">
+        <ion-text class="title-large settings-list-group__title title-h3">
+          {{ $msTranslate('SettingsModal.titles.display') }}
+        </ion-text>
+        <!-- change lang -->
+        <settings-option
+          class="settings-list__item"
+          :title="'SettingsModal.language.label'"
```

Comment:
You can remove all the `:title="'Text'"` while you're at it. `title="SettingsModal.language.label"` is cleaner.